src/components/backend-ai-resource-group-list.ts

```diff
@@ -888,6 +888,10 @@ export default class BackendAIResourceGroupList extends BackendAIPage {
                   label="${_t('resourceGroup.WsproxyAddress')}"
                   placeholder="http://localhost:10200"
                   value="${this.resourceGroupInfo?.wsproxy_addr ?? ''}"
+                  pattern="^(https?://)[a-zA-Z0-9]([a-zA-Z0-9-]*[a-zA-Z0-9])*.[a-zA-Z]{2,5}(:[0-9]{1,5})?(/.*)?$"
```

Comment:
If set the type of mwc-textfield to url, won't it automatically validate the url type if I just provide the autoValidate attribute? @agatha197

Reply:
😲